crates/node/p2p/src/discv5/driver.rs

```diff
@@ -314,6 +328,39 @@ impl Discv5Driver {
                             }
                         }
                     }
+                    event = event_stream.recv() => {
+                        let Some(event) = event else {
+                            trace!(target: "discovery", "Receiver `None` event");
+                            continue;
+                        };
+                        match event {
+                            discv5::Event::Discovered(enr) => {
+                                if EnrValidation::validate(&enr, chain_id).is_valid() {
+                                    debug!(target: "discovery", "Valid ENR discovered, forwarding to swarm: {:?}", enr);
+                                    self.store.add_enr(enr.clone());
+                                    let sender = enr_sender.clone();
+                                    tokio::spawn(async move {
+                                        if let Err(e) = sender.send(enr).await {
+                                            debug!(target: "discovery", "Failed to send enr: {:?}", e);
+                                        }
+                                    });
+                                }
+                            }
+                            discv5::Event::SessionEstablished(enr, addr) => {
+                                if EnrValidation::validate(&enr, chain_id).is_valid() {
+                                    debug!(target: "discovery", "Session established with valid ENR, forwarding to swarm. Address: {:?}, ENR: {:?}", addr, enr);
```

Comment:
```suggestion
                                    debug!(target: "discovery",
                                        %addr,
                                        ?enr,
                                        "Session established with valid ENR, forwarding to swarm"
                                    );
```
let's use the logs the way they were designed to be used with kv pairs. the above log will output
`.. addr=<the socket addr value> enr=<the enr in human readable format> ..`
NOTE! use the debug operator for enrs, ie `?enr`, otherwise they serialise in base 64 which is super unhelpful for debugging